src/public/js/product-page.js

```diff
@@ -0,0 +1,43 @@
+import Plyr from 'plyr';
+import ScrollMagic from 'scrollmagic';
+import '../../libs/vanilla-js-accordions/AccordionElement.min.js';
+
+new Plyr('#product-video');
+
+const scrollmagicController = new ScrollMagic.Controller();
+const sections = document.querySelectorAll('.product-info');
+const navLinks = document.querySelectorAll('.product-navigation__link-container');
+
+const updateActiveLink = (sectionId) => {
+  navLinks.forEach((link) => {
+    link.classList.remove('product-navigation__link-container--active');
+    const href = link.getAttribute('href');
+    if (href === `#${sectionId}`) {
+      link.classList.add('product-navigation__link-container--active');
+    }
+  });
+};
+
+sections.forEach((section, index) => {
+  let triggerHookValue = 0.5;
+  if (index === sections.length - 1) {
+    triggerHookValue = 0.7;
+  }
```

Comment:
> I see when you click over the description button, in some screen sizes (for example when you zoom out to 90% on chrome navigator), the button doesn´t change its color. Probably because the hook set on javascript doesn´t activate due the size and the color never changes. It needs more logic to solve it, but it let it as a comment. ![Captura de pantalla (1401)](https://user-images.githubusercontent.com/70463645/280877979-3ab08f28-3017-4dfd-8398-abdad84c8537.png)
> https://github.com/loopcrack-org/Kinub/pull/87#pullrequestreview-1716542142

![image](https://github.com/loopcrack-org/Kinub/assets/24960025/fa65597d-42b5-4f5a-9f79-39aba5430476)

@Mauriciocr207 this is not a bug. This is how the scroll magic plugin works, basically there is a invisible division in the viewport (50%) so whenever the purple line or the specific top section reaches the green line, automatically trigger a new tab change. 

We can fix that bug adding this code:

```suggestion
  const mediaQuery = window.matchMedia('(min-width: 1920px)');
  let triggerHookValue = 0.5;
  if (mediaQuery.matches) triggerHookValue = 0.3;
  if (index === sections.length - 1) {
    triggerHookValue = 0.7;
  }
```